### PR TITLE
Add support for mBuild's periodic polymer bonding

### PR DESCRIPTION
**Pull request by chrisjonesBSU**
The polymer builder in mBuild now lets you form periodic bonds, which basically just means you can bond the head and tail ports together. This should be useful for simulating polymer lattices where we want bonds formed across the the chain length direction.

This PR adds an option to the `Polymer` class that lets you use this built in feature in mBuild

### CODE REVIEW

File: flowermd/base/molecule.py [hunk 1]
```diff
@@ -477,6 +477,11 @@ class Polymer(Molecule):
         The bond length between connected atoms (units: nm)
     bond_orientation: list, default None
         The orientation of the bond between connected atoms.
+    periodic_bond_axis : str, default None
+        Axis along which to orient the polymer backbone along.
```
Comment (marjanalbooyeh) [inline flowermd/base/molecule.py:481]: two "along" in this sentence

File: flowermd/base/molecule.py [hunk 2]
```diff
@@ -513,14 +520,20 @@ def monomer(self):
...
         chain.add_monomer(
             self.monomer,
             indices=self.bond_indices,
             separation=self.bond_length,
             orientation=self.bond_orientation,
         )
-        chain.build(n=length, sequence="A")
+        chain.build(n=length, sequence="A", add_hydrogens=add_hydrogens)
+        if self.periodic_bond_axis:
+            chain.create_periodic_bond(axis=self.periodic_bond_axis)
```
Comment (marjanalbooyeh) [inline flowermd/base/molecule.py:536]: Maybe we can run a quick validation on `self.periodic_bond_axis` to make sure its value is valid (i.e. "x", "y", or "z") before calling the periodic bond function.

**Review by marjanalbooyeh — APPROVED** [review]
LGTM. Thank you Chris.
I just added two minor comments.

File: flowermd/base/molecule.py (hunk 2 above)
Comment (chrisjonesBSU) [reply flowermd/base/molecule.py:543]: Good idea, this should be in there now.

**Pull request merged**